### PR TITLE
Build scripts - Enables building to a mod target

### PR DESCRIPTION
**Pull request by Spoffy**

### CODE REVIEW

File: game/functions/systems/sites_hints/client/fn_sites_hints_glintJob.sqf [hunk 1]
```diff
@@ -22,7 +22,8 @@
 #define RADIUS 200
 #define OBJECTS_MAX 50
 
-private _interval = linearConversion [0, 1, vgm_c_skill_investigate_intensity, GLINT_JOB_INT_MAX, GLINT_JOB_INT_MIN];
+private _intensity = missionNamespace getVariable ["vgm_c_skill_investigate_intensity", 0];
+private _interval = linearConversion [0, 1, _intensity, GLINT_JOB_INT_MAX, GLINT_JOB_INT_MIN];
```
Comment (veteran29) [inline game/functions/systems/sites_hints/client/fn_sites_hints_glintJob.sqf]: This change is not needed if we init the global in preInit

File: mod/server/.hemtt/launch.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+parameters = [
+    "-skipIntro",
+    "-noSplash",
+    "-showScriptErrors",
+    "-debug",
+]
+executable = "arma3server_x64" # Default: "arma3_x64"
+file_patching = true
+binarize = false
+rapify = false
```
Comment (veteran29) [inline mod/server/.hemtt/launch.toml]: Any reason we do not want to rapify configs?

File: mod/server/.hemtt/launch.toml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+workshop = [
+    "1649778181" # VN testing build
+]
+dlc = []
+optionals = []
+parameters = [
+    "-skipIntro",
+    "-noSplash",
+    "-showScriptErrors",
+    "-debug",
```
Comment (veteran29) [inline mod/server/.hemtt/launch.toml]: These are default parameters.

File: mod/server/.hemtt/launch.toml (hunk 2 above)
Comment (Spoffy) [reply mod/server/.hemtt/launch.toml]: Not particularly, we can turn that on.

**Review by veteran29 — APPROVED** [review]
LGTM

**Pull request merged**